Modules/Package.swift

```diff
@@ -339,29 +361,12 @@ enum XcodeSupport {
             .xcodeTarget(
                 XcodeTargetNames.wooCommerceWatchApp,
                 dependencies: [
+                    "WooFoundationCore",
                     .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack"),
                     .product(name: "KeychainAccess", package: "KeychainAccess"),
                     .product(name: "Sentry", package: "sentry-cocoa"),
                 ]
             ),
-            .xcodeTarget(
-                XcodeTargetNames.wooFoundation,
-                dependencies: [
-                    "Codegen",
-                    .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack")
-                ]
-            ),
-            .xcodeTarget(
-                XcodeTargetNames.wooFoundationTests,
-                dependencies: ["TestKit", XcodeTargetNames.wooFoundation.asDependency]
-            ),
-            .xcodeTarget(
-                XcodeTargetNames.wooFoundationWatchOS,
-                dependencies: [
-                    "Codegen",
-                    .product(name: "CocoaLumberjackSwift", package: "CocoaLumberjack")
-                ]
-            ),
```

Comment:
As promised in #15654 , these `XcodeTarget_...` are beginning to disappear in favor of packages. The only ones that will remain are those for the apps and extensions targets.